insights/parsers/vdsm_log.py

```diff
@@ -156,3 +156,59 @@ def parse_lines(self, lines):
                     except:
                         pass
                 yield fields
+
+
+@parser(Specs.vdsm_import_log)
+class VDSMImportLog(LogFileOutput):
+    """Parser for the log file detailing virtual machine imports.
+
+    Sample log file::
+
+        [    0.2] preparing for copy
+        [    0.2] Copying disk 1/1 to /rhev/data-center/958ca292-9126/f524d2ba-155a/images/502f5598-335d-/d4b140c8-9cd5
+        [    0.0] >>> source, dest, and storage-type have different lengths
+
+    Examples::
+
+        >>> log = vdsm_import_logs.get('preparing for copy')
+        >>> len(log)
+        1
+        >>> log[0].get('raw_message', None)
+        '[    0.2] preparing for copy'
+        >>> log[0].get('vm_uuid', None)              # file: import-1f9efdf5-2584-4a2a-8f85-c3b6f5dac4e0-20180130T154807.log
+        '1f9efdf5-2584-4a2a-8f85-c3b6f5dac4e0'
+        >>> log[0].get('timestamp', None)
+        datetime.datetime(2018, 1, 30, 15, 48, 07)
+    """
+    time_format = '%Y%m%dT%H%M%S'
```

Comment:
This `self.time_format` is used for the `get_after` function inherited from the base class. It should be the format of the timestamp printed in the lines of the log. Please remove it and re-implement the `get_after` by referring to the [DmesgLineList's `get_after`](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/dmesg.py#L65)

Reply:
@xiangce Correct. I have removed `self.time_format` in update commit. Also as `timestamp` attribute is specific to file and not for each line, we not need to implement `get_after`, right?

Reply:
If you dod not implement it, the version defined in the base class will be inherited. It's still a callable method of this sub-class. And ERROR will be thrown when calling it. As I said you need to re-implement it as per the [DmesgLineList's `get_after`](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/dmesg.py#L65), otherwise, re-implement it by a single `pass` and explain it in the docstring.

Reply:
Done. (https://github.com/RedHatInsights/insights-core/pull/1707/commits/9251595d1e8a46b10ef6b7eaa2fcf6155c7b5f48)